spec/system/documents/edit_documents_spec.rb

```diff
@@ -30,6 +30,8 @@
     it "with valid data" do
       click_link "Edit"
 
+      expect(page).to have_content("If you add one or more numbers to this document, it will be published in the decision notice")
```

Comment:
This is the tiniest thing, but are we actually testing anything with this assertion? There's no logic to this text being there, it will always be there.

Reply:
That's true. I figured as it was specified in the A/Cs that we need this text on the Edit page, I should have a test for it. But then it's not likely to be something that intermittently disappears, so maybe a waste of an assertion. I will remove